tests/unit/test_kernel.py

```diff
@@ -285,6 +304,40 @@ def test_slab_major_consumers(self):
         self.assertEqual(top5, expected)
 
 
+class TestKernelNetworkInfo(TestKernelBase):
+    @utils.create_data_root({'proc/net/sockstat': PROC_SOCKSTAT})
+    def test_sockstat_parse(self):
+        uut = ProcSockStat()
+        self.assertEqual(uut.NsTotalSocksInUse, "908")
```

Comment:
as mentioned above, integer values should be saved as int() so that we can compare them with integers (not strings).

Reply:
done